ci-operator/step-registry/hypershift/azure/create/hypershift-azure-create-chain.yaml

```diff
@@ -15,7 +15,9 @@ chain:
       documentation: "Release image used for the HostedCluster. Empty by default it will use release:latest"
     - name: HYPERSHIFT_AZURE_LOCATION
       default: "eastus"
```

Comment:
We need to change the default value here, too, right?

Reply:
I believe not. The dev team utilizes the hypershift cluster profile, which, unlike azure-qe, does not provide a meaningful value for `$LEASED_RESOURCE`. Keeping this value is necessary for the existing jobs set up by the dev team to function correctly.